skore/tests/unit/sklearn/plot/precision_recall_curve/test_comparison_cross_validation.py

```diff
@@ -218,3 +218,53 @@ def test_multiclass_classification_kwargs(pyplot, multiclass_classification_repo
 
     display.plot(despine=False)
     assert display.ax_[0].spines["top"].get_visible()
+
+
+@pytest.mark.parametrize("with_average_precision", [False, True])
+def test_frame_binary_classification(
+    binary_classification_report, with_average_precision
+):
+    """Test the frame method with binary classification comparison cross-validation
+    data."""
+    report = binary_classification_report
+    display = report.metrics.precision_recall()
+
+    df = display.frame(with_average_precision=with_average_precision)
+    expected_index = ["estimator_name", "split_index"]
+    expected_columns = ["threshold", "precision", "recall"]
+    if with_average_precision:
+        expected_columns.append("average_precision")
+
+    check_frame_structure(df, expected_index, expected_columns)
+    assert df["estimator_name"].nunique() == len(report.reports_)
+
+    if with_average_precision:
+        for (_, _), group in df.groupby(["estimator_name", "split_index"]):
+            assert group["average_precision"].nunique() == 1
+
+
+@pytest.mark.parametrize("with_average_precision", [False, True])
+def test_frame_multiclass_classification(
+    multiclass_classification_report, with_average_precision
+):
+    """Test the frame method with multiclass classification comparison cross-validation
+    data."""
+    report = multiclass_classification_report
+    display = report.metrics.precision_recall()
+
+    df = display.frame(with_average_precision=with_average_precision)
+    expected_index = ["estimator_name", "split_index", "label"]
+    expected_columns = ["threshold", "precision", "recall"]
+    if with_average_precision:
+        expected_columns.append("average_precision")
+
+    check_frame_structure(df, expected_index, expected_columns)
+    assert df["estimator_name"].nunique() == len(report.reports_)
+    assert df["split_index"].nunique() == report.reports_[0]._cv_splitter.n_splits
+    assert df["label"].nunique() == len(
+        report.reports_[0].estimator_reports_[0].estimator_.classes_
+    )
```

Comment:
Let's include that in the constructor test because there is corner case to test.

```suggestion
```